plugins/Config/plugin.py

```diff
@@ -213,9 +213,9 @@ def _getValue(self, irc, msg, group, addChannel=False):
 
     def _setValue(self, irc, msg, group, value):
         if isReadOnly(group._name):
-            irc.error(_('This configuration variable is not writeable '
-                'via IRC. To change it you have to: 1) use @flush 2) edit '
-                'the config file 3) use @config reload.'), Raise=True)
+            irc.error(_("This configuration variable is not writeable "
+                "via IRC. To change it you have to: 1) use 'flush' 2) edit "
+                "the config file 3) use 'config reload'."), Raise=True)
```

Comment:
And I would 1) flush 2) config flush false 3)edit the config file 4)config reload.

I am on mobile web and I think I failed with commenting at least partially and I don't know what is the proper way to document this or would this be too detailed.

I also think that people aren't supposed to edit the config so I feel unsure about telling them to do that.